### PR TITLE
202 add flowers tutorial pw

### LINKED ISSUE #202: Add flowers tutorial
Add a simple tutorial for the flowers dataset including:
- Offer the dataset for download via a link (images + embedding)
- Show the user how to upload the dataset using the frontend (show alternatively how to upload it through the pip package)
- Show how to use the embedding/ sampling functionality in the web app

Name this tutorial "tutorial 2" and rename the active-learning tutorial to "tutorial 3"

### PR DESCRIPTION
**Pull request by philippmwirth**
# Sunflowers tutorial for uploading images & embeddings

Closes #202.

I added a quick tutorial on how to upload images and embeddings. The uploaded images and embeddings are used to create a sampling and explain the 2d scatter plot. 

I also added a download button for the sunflowers dataset.

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -34,13 +34,22 @@ download:
...
 	wget -N https://storage.googleapis.com/datasets_boris/pizzas.zip -P $(DATADIR)
 	unzip $(ZIPOPTS) $(DATADIR)/pizzas.zip  -d $(PLATFORMSOURCE)
 	mkdir -p $(PLATFORMSOURCE)/pizzas/margherita
 	mkdir -p $(PLATFORMSOURCE)/pizzas/salami
 	mv $(PLATFORMSOURCE)/pizzas/margherita*.jpg $(PLATFORMSOURCE)/pizzas/margherita
 	mv $(PLATFORMSOURCE)/pizzas/salami*.jpg $(PLATFORMSOURCE)/pizzas/salami
 
+	# sunflowers dataset
+	wget -N https://storage.googleapis.com/datasets_boris/Sunflowers.zip -P $(DATADIR)
+	unzip $(ZIPOPTS) $(DATADIR)/pizzas.zip  -d $(PLATFORMSOURCE)
```
Comment (IgorSusmelj) [inline docs/Makefile:47]: Are u sure about unzipping Pizzas again?

File: docs/source/tutorials_source/platform/tutorial_sunflowers.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+   create a new dataset. `Head over there and try it! <https://app.lightly.ai>`_
+
+
+Create a Sampling
+------------------
+
+Now, you have everything you need to create a sampling of your dataset. For this,
+head to the *Embedding* page of your dataset. You should see a two-dimensional
+scatter plot of your embeddings. If you hover over the images, their thumbnails
+will appear. Can you find clusters of similar images?
```
Comment (IgorSusmelj) [inline docs/source/tutorials_source/platform/tutorial_sunflowers.py:61]: I would add a screenshot here of the embedding screen

File: docs/source/tutorials_source/platform/tutorial_sunflowers.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+create a sampling. You will need to configure the following settings:
+
+* **Embedding:** Choose the embedding to use for the sampling.
+* **Sampling Strategy:** Choose the sampling strategy to use. This will be one of:
+
+   * Coreset: Greedily selects samples which are diverse.
+   * Coral: Combines Coreset with uncertainty scores to do active learning.
+   * Random: Selects samples uniformly at random.
+* **Stopping Condition:** Indicate how many samples you want to keep.
+* **Name:** Give your sampling a name. A new tag will be created under this name.
```
Comment (IgorSusmelj) [inline docs/source/tutorials_source/platform/tutorial_sunflowers.py:79]: I would add a screenshot here of the filled out "create sampling pop up"

**Review by IgorSusmelj — CHANGES_REQUESTED** [review]
A few small changes

File: docs/Makefile (hunk 1 above)
Comment (philippmwirth) [reply docs/Makefile:47]: No, good catch

**Review by IgorSusmelj — APPROVED** [review]

**Pull request merged**